src/app/core/auth/interceptors/jwt-interceptor.service.ts

```diff
@@ -0,0 +1,23 @@
+import { inject } from '@angular/core'
+import { HttpHandlerFn, HttpInterceptorFn, HttpRequest } from '@angular/common/http'
+import { environment } from 'src/environments/environment'
+import { Store } from '@ngxs/store'
+import { AuthStateSelectors } from "../state/auth.selectors";
+
+export const jwtInterceptor: HttpInterceptorFn = (
```

Comment:
Wrong function name. Functions *do something*, so they should be named as verbs.